go/vt/wrangler/reparent.go

```diff
@@ -498,22 +498,21 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 		wgSlaves.Wait()
 		return fmt.Errorf("failed to PopulateReparentJournal on master: %v", masterErr)
 	}
+	// Wait for the slaves to complete.
```

Comment:
making this change breaks one of the integ tests:
https://github.com/vitessio/vitess/blob/master/test/reparent.py#L572

We end up with this error:
```TopologyServer has inconsistent state for shard master```
because we have updated the masterElect to be master, but not updated the shard record.

cc: @enisoc

Reply:
Yeah that test case raises a good point. We can't require slaves to complete in order to consider the reparent finished, because we need to be able to make progress even if some slaves are down.